main.py

```diff
@@ -1,17 +1,25 @@
-import boto3
 import argparse
 import logging
 
 from AwsResources import AwsResources
+from Ssh import Ssh
+from rsync import perform_rsync
 
 parser = argparse.ArgumentParser(description='Load data on an AWS EBS volume')
```

Comment:
Sometimes the `help` strings begin with capital letters, sometimes they don't.

Reply:
Now the description of the program has a capital letter, all the descriptions do not